CHANGELOG.rst

```diff
@@ -9,6 +9,7 @@ v3.0.0-beta.3 (not yet released)
 
 *Added*
 
+- Oscillating pair potential for MD in ``hoomd.md.pair.OPP``.
```

Comment:
Please don't modify the change log in PRs. It almost guarantees conflicts which then requires that PRs modifying the change log need to be merged one at a time resolving conflicts in between. I update the change log with the message from the PR description directly on the master branch.

Reply:
My bad. I forget what packages to do this for.